### PR TITLE
Fixed examples

### LINKED ISSUE #294: inconsistencies found in location_example.json
**First one, this is incorrect:**
`voltage` found instead of `max_voltage` in location_example.json
Same goes for `amperage` and `max_amperage`.
This is not according to the documentation I think.

**Second one, this is inconsistent:**
`related_locations` is an empty list and is omitted. (This is allowed)
`status_schedule` is an empty list and became `[]`. (This is allowed)
I think a system would handle it for both things in the same way.

What exactly is leading? The docs or the examples? I think the docs?

### LINKED ISSUE #295: parking_restrictions not an array in some examples
I see this in some some location*.json examples:

` "parking_restrictions": "CUSTOMERS",`

but I expect:

` "parking_restrictions": ["CUSTOMERS"], `

because it's a list.

### LINKED ISSUE #296: typos? in cdr_locations.json example file
`"evses_uid": "3256",`

should be:

`"evse_uid": "3256",`

Also these shouldn't be plural connector**s**:

```
    "connectors_id": "1",
    "connectors_standard": "IEC_62196_T2",
    "connectors_format": "SOCKET",
    "connectors_power_type": "AC_1_PHASE"
```

### LINKED ISSUE #299: Example tariff_11_not_possible_alt_text.json has an invalid date
This is not right: `2018-17-29T15:55:58Z`

### LINKED ISSUE #300: Example tariff_7_first_hour_kwh_free.json is missing 'last_updated'
Title says it all.

### LINKED ISSUE #301: Example transport_and_format_version_details_example.json missing property 'role'
Subject says it all ;-)

### LINKED ISSUE #302: Example transport_and_format_get_token_[list_]example.json missing a lot of properties
See subject. Applies to two example files.

E.g. `country_code`.

### PR DESCRIPTION
**Pull request by frankhommers**
This fixes #302, fixes #301, fixes #300, fixes #299, fixes (partially) #297, fixes #296, fixes #295, fixes #294.

### CODE REVIEW

**Comment by RobertdeLeeuw-IHomer** [comment]
@frankhommers First of all, thank you for all the fixes!!! Great job.

Please read https://github.com/ocpi/ocpi/blob/master/CONTRIBUTING.md
Did you branch of a bugfix branch?
Could you change the merge request to the same bugfix branch?

**Comment by RobertdeLeeuw-IHomer** [comment]
Sorry for not having had the time to look at your pull request earlier, I have been busy with other projects.

File: examples/location_example.json [hunk 1]
```diff
@@ -59,14 +59,11 @@
       "amperage": 16,
       "tariff_ids": ["12"],
       "last_updated": "2015-06-29T20:39:09Z"
-    }],
-    "physical_reference": "2",
-    "floor_level": "-2",
-    "last_updated": "2015-06-29T20:39:09Z"
-  }],
```
Comment (RobertdeLeeuw-IHomer) [inline examples/location_example.json:66]: Why do you delete these fields? They are valid. Or do I miss the reason why you deleted them.

File: examples/location_example_parking_garage_opening_hours.json [hunk 2]
```diff
@@ -17,15 +17,17 @@
...
     "status": "AVAILABLE",
-    "connectors": [{
-      "id": "1",
-      "standard": "IEC_62196_T2",
-      "format": "SOCKET",
-      "power_type": "AC_3_PHASE",
-      "voltage": 230,
-      "amperage": 32,
-      "last_updated": "2017-03-07T02:21:22Z"
-    }],
```
Comment (RobertdeLeeuw-IHomer) [inline examples/location_example_parking_garage_opening_hours.json:28]: Is this only a different layout of the JSON? If we do this, we should do this for all the examples. 
I hope to generate examples in a next version so the layout is the same everywhere. 
So for now I would propose to leave them as is. Or did you change it in all the examples?

File: examples/location_example_uc3_destination_charger_not_published.json [hunk 3]
```diff
@@ -16,16 +16,18 @@
...
-      "id": "1",
-      "standard": "IEC_62196_T2",
-      "format": "SOCKET",
-      "power_type": "AC_3_PHASE",
-      "voltage": 220,
-      "amperage": 16,
-      "last_updated": "2019-07-01T12:12:11Z"
-    }],
-    "parking_restrictions": "CUSTOMERS",
+    "connectors": [
```
Comment (RobertdeLeeuw-IHomer) [inline examples/location_example_uc3_destination_charger_not_published.json:19]: Same as the other examples, this only changes the layout.

I do agree with the "parking_restrictions", that is a good find!!

File: examples/tariff_7_first_hour_kwh_free.json [hunk 4]
```diff
@@ -1,54 +1,71 @@
...
-  "elements" : [ {
-    "price_components" : [ {
-      "type" : "PARKING_TIME",
-      "price" : 0.0,
-      "step_size" : 60
-    } ],
-    "restrictions" : {
-      "min_duration" : 0,
-      "max_duration" : 3600
+  "id": "52",
```
Comment (RobertdeLeeuw-IHomer) [inline examples/tariff_7_first_hour_kwh_free.json:4]: Also in this example, most of this is only a layout change.

Only thing added to the file is: "last_updated": "2018-12-29T15:55:58Z"
Which is a good find.

**Review by RobertdeLeeuw-IHomer — COMMENTED** [review]
Thank you for this pull request.

Only comments in the layout changes to the JSON files.
I would prefer not to make these if not really needed as they are not inline with the rest of the specification. If you want I can fix these and then merge this.

File: examples/location_example_parking_garage_opening_hours.json (hunk 2 above)
Comment (frankhommers) [reply examples/location_example_parking_garage_opening_hours.json:28]: No it's amperage to max_amperage and voltage to max_voltage.

File: examples/location_example_uc3_destination_charger_not_published.json (hunk 3 above)
Comment (frankhommers) [reply examples/location_example_uc3_destination_charger_not_published.json:19]: No it's also amperage to max_amperage and voltage to max_voltage.

File: examples/tariff_7_first_hour_kwh_free.json (hunk 4 above)
Comment (frankhommers) [reply examples/tariff_7_first_hour_kwh_free.json:4]: Added and fixed the date from month 17 to 12.

**Pull request merged**